database/factories/OrderFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Order;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Order>
+ */
+class OrderFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'order_date' => $this->faker->dateTimeThisYear(),
+            'email' => $this->faker->safeEmail(),
+            'lid_name' => $this->faker->name(),
+            'user_id' => User::inRandomOrder()->first()->id,
+            'order_status_id' => $this->faker->numberBetween(1, 6),
+            'group_id' =>  $this->faker->numberBetween(1, 4)
```

Comment:
Dit zo hardcoded hierin zetten is niet uitbreidbaar, maar dit zal sws anders zijn als je voor order status een enum gebruikt. Voor de groep moet dit wel anders